tests/scanner/audit/iam_rules_engine_test.py

```diff
@@ -1405,6 +1403,87 @@ def test_project_required(self):
         ])
         self.assertItemsEqual(expected_violations, actual_violations)
 
+    def test_policy_all_projects_must_have_owners_from_domain(self):
+        """Test a policy where the owner belongs to the required domain.
+
+        Setup:
+            * Create a Rules Engine
+            * Create the policy bindings.
+            * Created expected violations list.
+
+        Expected results:
+            No policy violations found.
+        """
+        self.mock_org_rel_dao.find_ancestors = mock.MagicMock(
+            side_effect=[self.org789])
+
+        # actual
+        rules_local_path = get_datafile_path(__file__, 'test_rules_1.yaml')
+        rules_engine = ire.IamRulesEngine(rules_local_path)
+        rules_engine.rule_book = ire.IamRuleBook(
+            {}, test_rules.RULES11, self.fake_timestamp)
+        rules_engine.rule_book.org_res_rel_dao = mock.MagicMock()
+        find_ancestor_mock = mock.MagicMock(
+            side_effect=[[self.org789]])
+        rules_engine.rule_book.org_res_rel_dao.find_ancestors = \
+            find_ancestor_mock
+
+        policy = {
+            'bindings': [{
+                'role': 'roles/owner',
+                'members': ['user:def@xyz.edu']
+            }]
+        }
+
+        actual_violations = set(rules_engine.find_policy_violations(
+            self.project1, policy))
+
+        # expected
+        expected_violations = set()
+
+        self.assertEqual(expected_violations, actual_violations)
+
+    def test_policy_all_projects_must_have_owners_from_wildcard_domain(self):
```

Comment:
Please clarify this method name, to show that it's using the wildcard domain from user type.  So, a good name would be:
```
test_policy_all_projects_must_have_owners_from_wildcard_domain_of_user_type
```